src/operations/blas1/WGAtomicReduction.hpp

```diff
@@ -63,11 +63,11 @@ template <typename operator_t, bool usmManagedMem, typename lhs_t,
 PORTBLAS_INLINE
     typename WGAtomicReduction<operator_t, usmManagedMem, lhs_t, rhs_t>::value_t
     WGAtomicReduction<operator_t, usmManagedMem, lhs_t, rhs_t>::eval(
-        cl::sycl::nd_item<1> ndItem) {
+        sycl::nd_item<1> ndItem) {
   auto atomic_res =
-      cl::sycl::atomic_ref<value_t, cl::sycl::memory_order::relaxed,
-                           cl::sycl::memory_scope::device,
-                           cl::sycl::access::address_space::global_space>(
+      sycl::atomic_ref<value_t, sycl::memory_order::relaxed,
+                           sycl::memory_scope::device,
+                           sycl::access::address_space::global_space>(
```

Comment:
In some places the code formatting seems quite odd.

Reply:
I reformatted everything again using `git-clang-format` in 1e50d9f now it should be fine